packages/algob/src/types.ts

```diff
@@ -341,14 +343,29 @@ export interface Checkpoint {
 export type ASADef = z.infer<typeof ASADefSchema>;
 export type ASADefs = z.infer<typeof ASADefsSchema>;
 
-export interface ASADeploymentFlags {
-  creator: Account
+export interface DeploymentFlags {
   totalFee?: number
-  feePerByte?: number
   firstValid?: number
   validRounds?: number
 }
 
+export interface ASADeploymentFlags extends DeploymentFlags {
+  creator: Account
+  feePerByte?: number
```

Comment:
Consult with Robert to use this parameter.

Reply:
This parameter should be used in the same way as in ASA flags. And then you have to move it to `DeploymentFlags`.

Reply:
This was not in specs , there is only one fee/flat  fee in specs

Reply:
what do i do ? include it or not ?

Reply:
Done

Reply:
In the spec this is under:
```
  payTxParams: Object // other raw payment transaction parameters 
```

This has to be compatible as with all other transactions.

Reply:
Yes, it is same for both for paymentParams